scripts/image-builder/build.sh

```diff
@@ -373,15 +396,16 @@ if [ -n "${CUSTOM_RPM_FILES}" ] ; then
 fi
 
 title "Loading sources for OpenShift and MicroShift"
-for f in openshift-local microshift-local custom-rpms ; do
+for f in microshift-local custom-rpms ; do
     [ ! -d ${f} ] && continue
     sed "s;REPLACE_IMAGE_BUILDER_DIR;${BUILDDIR};g" "${SCRIPTDIR}/config/${f}.toml.template" > ${f}.toml
     sudo composer-cli sources delete ${f} 2>/dev/null || true
     sudo composer-cli sources add "${BUILDDIR}/${f}.toml"
 done
 
 title "Preparing blueprints"
-cp -f "${SCRIPTDIR}"/config/{blueprint_v0.0.1.toml,installer.toml} .
+cp -f "${SCRIPTDIR}"/config/installer.toml .
+envsubst <"${SCRIPTDIR}"/config/blueprint_v0.0.1.toml >blueprint_v0.0.1.toml
```

Comment:
`envsubst` is definitely a better techinque than sed replacement. However, I think we'd better convert all the `sed` calls to `envsubst` in the same PR. At this time, can we keep one consistent technique for replacements?
In any case, there is a convention in the script that uses `REPLACE_` prefix for all such variables. It should help us avoid unwanted replacements from the user environment.